microsoft/testsuites/ltp/ltp.py

```diff
@@ -52,30 +54,58 @@ class Ltp(Tool):
     LTP_DIR_NAME = "ltp"
     DEFAULT_LTP_TESTS_GIT_TAG = "20230929"
     LTP_GIT_URL = "https://github.com/linux-test-project/ltp.git"
+    LTP_RESULT_FILE = "ltp-results.log"
+    LTP_OUTPUT_FILE = "ltp-output.log"
+    LTP_SKIP_FILE = "skipfile"
     BUILD_REQUIRED_DISK_SIZE_IN_GB = 2
-    LTP_RESULT_PATH = "/opt/ltp/ltp-results.log"
-    LTP_OUTPUT_PATH = "/opt/ltp/ltp-output.log"
-    LTP_SKIP_FILE = "/opt/ltp/skipfile"
     COMPILE_TIMEOUT = 1800
     RUN_TIMEOUT = 12000
 
     @property
     def command(self) -> str:
-        return "/opt/ltp/runltp"
+        return self._command_path + "runltp"
```

Comment:
Use f string like `"{self._command_path}runltp"`

Reply:
The change is done, thanks